apps/articles/client/articles.coffee

```diff
@@ -3,49 +3,86 @@ _ = require 'underscore'
 sd = require('sharify').data
 Backbone = require 'backbone'
 Articles = require '../../../collections/articles.coffee'
-PoliteInfiniteScrollView = require '../../../components/polite_infinite_scroll/client/view.coffee'
 EditorialSignupView = require './editorial_signup.coffee'
 articleTemplate = -> require('../templates/articles_feed.jade') arguments...
+request = require 'superagent'
+{ crop } = require '../../../components/resizer/index.coffee'
+{ toSentence } = require 'underscore.string'
 
-module.exports.MagazineView = class MagazineView extends PoliteInfiniteScrollView
-
-  initialize: ({@offset, @params})->
-    @collection.on 'sync', @onSync
-    @onInitialFetch()
+module.exports.MagazineView = class MagazineView extends Backbone.View
 
+  initialize: ({@offset})->
+    @$spinner = @$('#articles-page .loading-spinner')
     @$('.is-show-more-button').click => @startInfiniteScroll()
 
-  onInfiniteScroll: ->
+  onInfiniteScroll: =>
     return if @finishedScrolling
-    @collection.fetch
-      data: @params
-      remove: false
-      success: (articles, res) =>
-        @params.offset += 10
-        @onFinishedScrolling() if res.length is 0
+    @offset += 30
+    query = """
+      {
+        articles(published: true, limit: 30, sort: "-published_at", featured: true, offset: #{@offset} ) {
+          slug
+          thumbnail_title
+          thumbnail_image
+          tier
+          published_at
+          channel_id
+          author{
+            name
+          }
+          contributing_authors{
+            name
+          }
+        }
+      }
+    """
+    request.post(sd.POSITRON_URL + '/api/graphql')
```

Comment:
I know we use our own wrapper lib for Metaphysics, but might make sense to use a generic one now like [Lokka](https://github.com/kadirahq/lokka) to clean up these requests. Not a blocker, just making note.